switch-configuration/config/types/Booth

```diff
@@ -1,8 +1,6 @@
 // Expo Booth Area switch Template
-TRUNK	ge-0/0/0	exSCALE-SLOW,exSCALE-FAST,exSpeaker,exInfra,exAVLAN, \
-				exSigns,exVmVendor,vendor_backbone			-	Uplink
-TRUNK	ge-0/0/1	exSCALE-SLOW,exSCALE-FAST,exSpeaker,exInfra,exAVLAN, \
-				exSigns,exVmVendor,vendor_backbone			-	Downlink
+TRUNK	ge-0/0/0	exSCALE-SLOW,exSCALE-FAST,exSpeaker,exInfra,exAVLAN, exSigns,exVmVendor,vendor_backbone			-	Uplink
```

Comment:
Id prefer it we update `test_datafile` to support the `\` for multiline entries. Would you mind seeing how feasible that approach would be?